### PR TITLE
Small screen resolution precision

### LINKED ISSUE #13662: Small screen resolution and icon display
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[docs/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository)

### What part(s) of the article would you like to see updated?

Precision on branch icon being hidden with some resultions which makes content irrelebant and confusing

### Additional information

tried to write a fix, see #13659

### PR DESCRIPTION
**Pull request by mathieu-aubin**
It appears that the branch '{% octicon "git-branch" aria-label="The branch icon" %}' gets removed past a certain resolution and this tutorial, altho correct, becomes somewhat of a big question mark.

I have never had the issue happen until i started using this 15 inches NEC monitor SIDEWAYS as a second monitor on my current dev stack in order to test out somewhat accurately some odd resolutions mobiles have and their effects on browsing.

I couldn't figure out why i had been able to delete branches from the web gui for all i can remember and now, very very confusingly, couldnt figure out it for the life of me. Even so i had to google it because it killed my chi.

Decided to add to the docs, please feel free to do whatever you want with that and, thanks!

### Why:

caused an issue as described when using small resolution screens which hide the subject

Closes #13662

### What's being changed:

just a precision within a NOTE block

### Check off the following:

- [X] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [X] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by mathieu-aubin** [comment]
This is the result of command `xrandr` on the computer i'm using with regards to the monitor in question.

```Screen 0: minimum 320 x 200, current 2688 x 1080, maximum 16384 x 16384
VGA-1 connected 768x1024+1920+0 right (normal left inverted right x axis y axis) 304mm x 228mm
   1024x768      75.03 +  70.07    60.00* 
   832x624       74.55  
   800x600       72.19    75.00    60.32    56.25  
   640x480       75.00    72.81    66.67    59.94  
   720x400       70.08  
```
It is rotated to the right. and is part of a larger display server of (1920+768)=2688 px

Thanks!

**Comment by mathieu-aubin** [comment]
Could also mention that the thing can be accessible via url `/branches` within a project

File: content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ shortTitle: Create & delete branches
 
 **Note:** If the branch you want to delete is the repository's default branch, you must choose a new default branch before deleting the branch. For more information, see "[Changing the default branch](/github/administering-a-repository/changing-the-default-branch)."
 
+Also, as browsers and their interpretation of current standards can potentially deliver visually different results and you are having issues locating the {% octicon "git-branch" aria-label="The branch icon" %} icon, try zooming out some. It should show up and become usable as designed.
```
Comment (zbyte-king) [inline content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository.md]: Interpret

**Comment by ramyaparimi** [comment]
@mathieu-aubin 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by ethomson** [comment]
I think that we should fix this bug instead of documenting the workaround.  🤔  Let me pass this on to the engineering team.

**Comment by davidstaheli** [comment]
@mathieu-aubin, thanks for doing this! I agree with @ethomson above that we should treat this as a bug to fix so that the workaround wouldn't be necessary.

**Comment by skedwards88** [comment]
Thanks for weighing in! I'm going to close out this PR since we will fix the bug instead of documenting this, but we hope to see more contributions from you in the future @mathieu-aubin !

**Comment by mathieu-aubin** [comment]
Great. All i hoped for was some kind of resolution. But... is this fair to call this a BUG? I mean... it seems the site IS functionnal yet, the DOCS dont take the programming into consideration. Is this a bug or an omission, a mishap in writting docs, maybe not writting as universally as it best would be.

Dont get me wrong, i will, again, repeat; im all about resolution on issues that might pose new users or heck, all for any kind of resolution.

I really am puzzled with the fact that its called a bug - i dont see 'wording' or 'written explanations' being able to be affected by what we can call a bug (as in the moth flew into the glass tube amp and fucked it all).

I appreciate whatever input you have on my concern, no matter opinionated with regards to me being 'pulling hairs' or bein anal about a docs group being accurate.

That being said, its all 'write' with me, i wont loose sleep over it as i dont do much of that lately!

Cheers and may the shwartz be with j00

**Pull request closed without merging**